railties/lib/rails/generators/app_base.rb

```diff
@@ -326,8 +322,12 @@ def bundle_command(command)
 
         require 'bundler'
         Bundler.with_clean_env do
-          output = `"#{Gem.ruby}" "#{_bundle_command}" #{command}`
-          print output unless options[:quiet]
+          full_command = "\"#{Gem.ruby}\" \"#{_bundle_command}\" #{command}"
```

Comment:
It might be better to use `%Q[]` or something like that to build the string, to avoid escaping quotes.